### PR TITLE
feat(website): update contents on the website

### PR DESCRIPTION
**Pull request by SANTHSIM22**
- [x] Header
- [X] Hero
- [x] Footer

### CODE REVIEW

File: src/components/Navbar.astro [hunk 1]
```diff
@@ -1,21 +1,34 @@
...
-  { href: '/blog', label: 'Blog' },
+  { href: '/about', label: 'Home' },
+  { href: '/#services', label: 'About' },
+  { href: '/#projects', label: 'Services' },
+  { href: '/pricing', label: 'Event' },
+  { href: '/blog', label: 'Sponsors' },
 ];
 
 import Container from "./Container.astro"
+import Gdsclogo from "./Gdsclogo.astro";
```
Comment (anishshobithps) [inline src/components/Navbar.astro:15]: Please keep the logo name as GDSCLogo

File: src/components/Navbar.astro [hunk 2]
```diff
@@ -24,22 +37,22 @@ import Container from "./Container.astro"
...
             <li>
-                <a href={link.href} class="block py-2 px-3 text-black rounded md:bg-transparent md:p-0">{link.label}</a>
+                <a href={link.href} class="bebas-neue-regular text-black  hover:text-red block py-2 px-3 text rounded md:bg-transparent md:p-0 ">{link.label}</a>
             </li>
             ))}
             <li>
-                <a class="md:hidden px-9 py-5 rounded-2xl border border-zinc-900 bg-white hover:bg-black text-black hover:text-white inline-flex" href="https://github.com/manulthanura/Positivus" target="_blank">
-                  <h2 class="text-center text-xl font-normal leading-7">Request a quot</h2>
+                <a class="md:hidden">
+                  <img src="/image.png">
```
Comment (anishshobithps) [inline src/components/Navbar.astro]: See, if you can use Astro's Image class

File: src/components/Footer.astro [hunk 3]
```diff
@@ -1,24 +1,31 @@
...
+            <span class="icon-[skill-icons--instagram]" style="width: 30px; height: 50px; margin-top: 10px; margin-left: 10px;"></span>
+        </a>        
     </div>
-    </Container>
-</footer>
+            <div class="  flex flex-row items-center md:m-0 md: ml-8 ">
+                <Gdsclogo/>
+                <h1 class="roboto-bold-italic">Google Developer Student Club Mangaluru</h1>
 
-<script is:inline>
```
Comment (anishshobithps) [inline src/components/Footer.astro:21]: It's better to have All rights reserved right ?

**Review by anishshobithps — CHANGES_REQUESTED** [review]
Also, the Mangaluru Logo, use a bold font, like poppins it will look good.

File: src/components/Footer.astro (hunk 3 above)
Comment (SANTHSIM22) [reply src/components/Footer.astro:21]: okay i will add it

**Pull request closed without merging**